lib/modules/apostrophe-events-calendar-widgets/lib/assets.js

```diff
@@ -0,0 +1,7 @@
+module.exports = function(self, options) {
+  self.pushAssets = function() {
+    self.pushAsset('script', 'vendor/clndr.min', { when: 'always' });
```

Comment:
How about an option to not push this, in case they already have it? Wouldn't get used much I know.

Reply:
added options to omit pushing both `clndr` and `moment`. `moment` is only eligible to be pushed in a lean world but can still be passed an option to omit.